henchman/helpers.go

```diff
@@ -21,6 +24,16 @@ func MergeMap(src map[interface{}]interface{}, dst map[interface{}]interface{},
 	}
 }
 
+func MergeLogrusFields(src log.Fields, dst log.Fields, override bool) {
```

Comment:
I agree. a `mergeable` interface will be great. In case of log fields, we
can create a proxy struct that composes log.Fields. But thread safely as we
don't want to complicate things by 'refactoring'.

On Thu, Nov 12, 2015 at 5:33 PM, Madhurranjan Mohaan <
notifications@github.com> wrote:

> In henchman/helpers.go
> <https://github.com/apigee/henchman/pull/81#discussion_r44740990>:
>
> > @@ -21,6 +24,16 @@ func MergeMap(src map[interface{}]interface{}, dst map[interface{}]interface{},
> >  	}
> >  }
> >
> > +func MergeLogrusFields(src log.Fields, dst log.Fields, override bool) {
>
> Can we have a single merge method that takes interfaces ? (More
> refactoring ;) )
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apigee/henchman/pull/81/files#r44740990>.
>



-- 
- Sudharshan